sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto.ts

```diff
@@ -1,18 +1,19 @@
 import { RestrictionType } from "@sims/sims-db";
 
 /**
- * Base DTO for application restriction bypass.
+ * DTO class for AEST application restriction bypass summary.
```

Comment:
I believe the comment should not be client specific. `AEST` should be removed. @dheepak-aot please correct me if wrong.

Reply:
Yes unless there is a need(when there is an intention to use separate DTOs for different clients) the comments must not be client specific.

Reply:
Updated.